tools/act/plugin.yaml

```diff
@@ -0,0 +1,22 @@
+version: 0.1
+downloads:
+  - name: act
+    downloads:
+      - os:
+          linux: Linux
+          macos: Darwin
+        cpu:
+          x86_64: x86_64
+          arm_64: arm64
+        url: https://github.com/nektos/act/releases/download/v${version}/act_${os}_${cpu}.tar.gz
+      - os:
+          windows: Windows
+        cpu:
+          arm_64: arm64
```

Comment:
They don't have an x86_64 download?

Reply:
not for that version - added a version specifier